core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -1339,6 +1342,20 @@ public static boolean useLegacyRouteDecoratorOrdering() {
         return DEFAULT_USE_LEGACY_ROUTE_DECORATOR_ORDERING;
     }
 
+    /**
+     * Returns whether to allow double dots ({@code ..}) in query string.
+     *
+     * <p>Note that double dots pattern could lead to a vulnerability if a query param value contains
+     * an improper path such as {@code /download?path=../../secrets.txt}, so you should carefully enable
```

Comment:
, so ... -> . Therefore, extra caution should be taken when enabling this option, and you may need ...